api/optimeet/groups/views.py

```diff
@@ -5,22 +5,31 @@
 from django.views.decorators.csrf import csrf_exempt
 from rest_framework.parsers import JSONParser
 from rest_framework import status
+from django import db
 
+from .models import rnd_id
 from . import models
 from . import serializers
 
 @csrf_exempt
 @api_view(['POST'])
 def groups(request):
     
-    serializer = serializers.GroupSerializer(data = request.data)
+    get_id = rnd_id()
+
+    request_data = request.data.copy()
+    id = {'group_id':get_id}
+    request_data.update(id)
+
+    serializer = serializers.GroupSerializer(data = request_data)
     
     if serializer.is_valid():
         serializer.save()
+        
         return Response(serializer.data, status=status.HTTP_201_CREATED)
-    
+            
     return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
-
+    
 @csrf_exempt
 @api_view(['GET'])    
 def group_detail(request, group_id):
```

Comment:
This GET doesn't work anymore since we removed the `id` field. Be sure to test the endpoints your models are a part of whenever you make changes! To help you with this, it's very good practice to almost always have a section in your PR description where you mention the tests you've ran.

Reply:
fixed it